docs/concepts/storage/volumes.md

```diff
@@ -70,7 +70,6 @@ Kubernetes supports several types of Volumes:
    * `azureFile`
    * `cephfs`
    * `configMap`
-   * `csi`
```

Comment:
@saad-ali why are we removing this? I can understand the CSI plugin is out-of-tree, but from users' perspective, they are still called `csi` volumes, right?